core/src/main/java/org/apache/brooklyn/core/internal/BrooklynPropertiesImpl.java

```diff
@@ -439,24 +442,45 @@ public void putAll(Map vals) {
     }
 
     @Override
-    public Object getRawConfig(ConfigKey<?> key) {
-        return get(key.getName());
+    public Maybe<Object> getConfigRaw(ConfigKey<?> key) {
+        if (containsKey(key.getName())) return Maybe.of(get(key.getName()));
+        return Maybe.absent();
     }
     
     @Override
     public Maybe<Object> getConfigRaw(ConfigKey<?> key, boolean includeInherited) {
-        if (containsKey(key.getName())) return Maybe.of(get(key.getName()));
-        return Maybe.absent();
+        return getConfigRaw(key);
     }
 
     @Override
-    public Map<ConfigKey<?>, Object> getAllConfig() {
+    public Maybe<Object> getConfigLocalRaw(ConfigKey<?> key) {
+        return getConfigRaw(key);
+    }
+    
+    @Override
+    public Map<ConfigKey<?>,Object> getAllConfigLocalRaw() {
         Map<ConfigKey<?>, Object> result = new LinkedHashMap<ConfigKey<?>, Object>();
         for (Object entry: entrySet())
             result.put(new BasicConfigKey<Object>(Object.class, ""+((Map.Entry)entry).getKey()), ((Map.Entry)entry).getValue());
         return result;
     }
 
+    @Override @Deprecated
```

Comment:
When deprecating, can we include javadoc to say when and to direct people to the correct method:

```
/**
 * @deprecated since 0.10.0; instead use {@link #getAllConfigLocalRaw()}
 */
```